scripts/README.md

```diff
@@ -1,48 +1,102 @@
 # ComfyUI Helper Scripts
 
-This directory contains various helper scripts designed to streamline working with ComfyStream:
+This directory contains helper scripts to simplify the deployment and management of **ComfyStream**:
 
-- `spinup_comfystream_tensordock.py`: Spins up a ComfyStream instance on a [Tensordock server](https://tensordock.com/), installs dependencies, and starts the server.
+- **Ansible Playbook (`ansible/plays/setup_comfystream.yml`)** – Deploys ComfyStream on any cloud provider.  
+- **`spinup_comfystream_tensordock.py`** – Fully automates VM creation and ComfyStream setup on a [TensorDock server](https://tensordock.com/).  
 
 ## Usage Instructions
 
-### Spinup TensorDock ComfyStream Instance
+### Ansible Playbook (Cloud-Agnostic Deployment)
 
-1. **Create a Tensordock Account**: Register at [Tensordock](https://dashboard.tensordock.com/register), attach a credit card, and create an API key and token.
-2. **Set Up a Python Virtual Environment**:
-   Create and activate a virtual environment using [Conda](https://docs.anaconda.com/miniconda/) and install the required dependencies:
+This repository includes an [Ansible playbook](https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_intro.html) to deploy ComfyStream on any cloud provider. Follow the steps below:
+
+1. **Create a VM**:  
+   Deploy a VM on a cloud provider such as [TensorDock](https://marketplace.tensordock.com/deploy?gpu=geforcertx4090-pcie-24gb&gpuCount=1&ramAmount=16&vcpuCount=4&storage=100&os=Ubuntu-22.04-LTS), AWS, Google Cloud, or Azure with the following minimum specifications:
+   - **GPU**: 20GB VRAM  
+   - **RAM**: 16GB  
+   - **CPU**: 4 vCPUs  
+   - **Storage**: 100GB
+
+   > [!TIP]
+   > You can use the `spinup_comfystream_tensordock.py --bare-vm` script to create a [compatible VM on TensorDock](https://marketplace.tensordock.com/deploy?gpu=geforcertx4090-pcie-24gb&gpuCount=1&ramAmount=16&vcpuCount=4&storage=100&os=Ubuntu-22.04-LTS).
+
+2. **Open Required Ports**:  
+   Ensure the following ports are open **inbound and outbound** on the VM's firewall/security group:
+   - **SSH (Port 22)** – Remote access  
+   - **HTTPS (Port 8189)** – ComfyStream access  
+  
+3. **Install Ansible**:  
+   Follow the [official Ansible installation guide](https://docs.ansible.com/ansible/latest/installation_guide/index.html).
+
+4. **Configure the Inventory File**:
+   Add the VM’s public IP to `ansible/inventory.yml`.
+
+5. **Run the Playbook**:  
+   Execute:
 
     ```bash
-    conda create -n comfystream python=3.8
-    conda activate comfystream
-    pip install -r requirements.txt
+    ansible-playbook -i ansible/inventory.yaml ansible/plays/setup_comfystream.yaml
     ```
```

Comment:
If you're deploying with a non-sudo user account, it is necesary to use `--ask-become-pass` or configure a vault

Reply:
Good point will update the readme.